packages/flutter/lib/src/widgets/_web_image_io.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../../widgets.dart';
+
+/// Returns [true] if the bytes at [url] can be fetched.
+///
+/// The bytes may be unable to be fetched if they aren't from the same origin
+/// and the sever hosting them does not allow cross-origin requests.
+Future<bool> checkIfImageBytesCanBeFetched(String url) {
+  // On mobile, there are no cross-origin restrictions.
+  return Future<bool>.value(true);
+}
+
+/// Returns a widget which displays the [src] in an <img> tag.
+Widget createImgElementWidget(String src,
```

Comment:
Should this be `createImageElementWidget`, per https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#avoid-abbreviations

Just a suggestion, since `ImgElement` probably aligns with the `<img>` tag.

Reply:
With this name I am trying to make it clear that this creates a Widget backed by an `<img>` element.